config: better diagnostic for a corrupted snapshot (#9494)
tarantool/tarantool: changelogs/unreleased/config-better-corrupted-snap-diagnostic.md

## bugfix/config

> * Give a better diagnostic for a corrupted snapshot file: if it has no instance
>   or replicaset UUID (gh-8862).

Reviewer: ```suggestion
* Improved the diagnostic given for a corrupted snapshot file without an instance or replica set UUID (gh-8862).
```
